go.mod

```diff
@@ -45,7 +45,7 @@ require (
 	go.opentelemetry.io/otel/sdk v1.20.0
 	go.opentelemetry.io/otel/trace v1.20.0
 	golang.org/x/net v0.17.0
-	golang.org/x/oauth2 v0.11.0
+	golang.org/x/oauth2 v0.12.0
```

Comment:
Is this change required?

Reply:
nope, this was just for triggering the workflow (it only executes if there are changes to go.mod) but thought of keeping it so we can move the updates and see this being triggered in other PRs.